spec/results/proto-all-params-azure.yml

```diff
@@ -365,57 +412,49 @@ networks:
   type: manual
 releases:
 - name: bosh
-  sha1: 8d0c61e5cfaa1fc3c58a678ca700da1db8e8c567
-  url: https://bosh.io/d/github.com/cloudfoundry/bosh?v=271.0.0
+  sha1: 3810c75c8a2ffa61adbe80809f7a361f0bcf4cd9
+  url: https://s3.amazonaws.com/bosh-compiled-release-tarballs/bosh-271.0.0-ubuntu-xenial-621.71-20200422-220715-218184745-20200422220716.tgz
   version: 271.0.0
 - name: bpm
-  sha1: c956394fce7e74f741e4ae8c256b480904ad5942
-  url: https://bosh.io/d/github.com/cloudfoundry/bpm-release?v=1.1.8
+  sha1: d9a22dff24a659a8f6585a600fdd7e27bf211f3d
+  url: https://s3.amazonaws.com/bosh-compiled-release-tarballs/bpm-1.1.8-ubuntu-xenial-621.71-20200422-220353-760960221-20200422220355.tgz
   version: 1.1.8
 - name: uaa
-  sha1: ad4967615f2321648372497402ca6359ab8ba0df
-  url: https://bosh.io/d/github.com/cloudfoundry/uaa-release?v=74.17.0
+  sha1: 186514cb390f6be7eb39a6ce861c99b0dc87f80b
+  url: https://s3.amazonaws.com/bosh-compiled-release-tarballs/uaa-74.17.0-ubuntu-xenial-621.71-20200422-215929-649183041-20200422215932.tgz
   version: 74.17.0
 - name: credhub
-  sha1: c45af16ed393bb3cf061b8749e3ee4cae74ce995
-  url: https://bosh.io/d/github.com/pivotal-cf/credhub-release?v=2.6.0
+  sha1: 479420f5ec04945b5dc76db955588bcd7603ff01
+  url: https://s3.amazonaws.com/bosh-compiled-release-tarballs/credhub-2.6.0-ubuntu-xenial-621.71-20200422-215930-454454003-20200422215931.tgz
   version: 2.6.0
 - name: bosh-azure-cpi
   sha1: f658b25b6404a5b5676ab6379b99170f300a5124
   url: https://bosh.io/d/github.com/cloudfoundry/bosh-azure-cpi-release?v=37.2.0
   version: 37.2.0
 - name: os-conf
-  sha1: 7579a96515b265c6d828924bf4f5fae115798199
-  url: https://bosh.io/d/github.com/cloudfoundry/os-conf-release?v=21.0.0
-  version: 21.0.0
+  sha1: 6946056ad69ae378cb89c9ef76daf66370a7dc6a
+  url: https://bosh.io/d/github.com/cloudfoundry/os-conf-release?v=22.0.0
+  version: 22.0.0
 resource_pools:
 - cloud_properties:
-    availability_set: test-availability-set
-    ephemeral_disk:
-      use_root_disk: true
-    instance_type: test-instance-type
-    root_disk:
-      size: 27146
```

Comment:
Make sure `availability_set`, `ephemeral_disk`, `instance_type` and `root_disk` are still propagated

Reply:
is this going to be an feature? why would we use instance types?

Reply:
this is still supported but maby should be moved to migrated to use availability_zones in the future?